templates/common/_base/files/vsphere-disable-vmxnet3v4-features.yaml

```diff
@@ -7,12 +7,40 @@ contents:
       # https://bugzilla.redhat.com/show_bug.cgi?id=1941714
       # https://bugzilla.redhat.com/show_bug.cgi?id=1935539
       # https://bugzilla.redhat.com/show_bug.cgi?id=1987108
+      # https://issues.redhat.com/browse/SPLAT-1409
 
       driver=$(nmcli -t -m tabular -f general.driver dev show "${DEVICE_IFACE}")
 
-      if [[ "$2" == "up" && "${driver}" == "vmxnet3" ]]; then
+      if [[ "$2" != "up" ]]; then
+        exit
+      fi
+      
+      if [[ "${driver}" == "vmxnet3" ]]; then
         logger -s "99-vsphere-disable-tx-udp-tnl triggered by ${2} on device ${DEVICE_IFACE}."
         ethtool -K ${DEVICE_IFACE} tx-udp_tnl-segmentation off
         ethtool -K ${DEVICE_IFACE} tx-udp_tnl-csum-segmentation off
         ethtool -K ${DEVICE_IFACE} tx-checksum-ip-generic off
+        ethtool -K ${DEVICE_IFACE} tx-esp-segmentation off
+        ethtool -K ${DEVICE_IFACE} esp-hw-offload off
+        ethtool -K ${DEVICE_IFACE} esp-tx-csum-hw-offload off
```

Comment:
The base `vmxnet3` devices are always `off [fixed]`.  I'm not sure what the `[fixed]` means.

```
ethtool-k-bond0-192.168.221.35.log
tx-esp-segmentation: off
esp-hw-offload: off
esp-tx-csum-hw-offload: off

ethtool-k-ens192-192.168.221.35.log
tx-esp-segmentation: off [fixed]
esp-hw-offload: off [fixed]
esp-tx-csum-hw-offload: off [fixed]

```

Reply:
I interpret that as not configurable, so can't be set to `on`

Reply:
Maybe I should leave this part out because it will get noisy when it fails